pep-0590.rst

```diff
@@ -48,145 +53,204 @@ The function pointer type
 Calls are made through a function pointer taking the following parameters:
 
 * ``PyObject *callable``: The called object
-* ``Py_ssize_t n``: The number of arguments plus an optional offset flag for the first argument in vector.
-* ``PyObject **args``: A vector of arguments
+* ``Py_ssize_t n``: The number of arguments plus the optional flag ``PY_VECTORCALL_ARGUMENTS_OFFSET`` (see below)
+* ``PyObject *const *args``: A vector of arguments
 * ``PyTupleObject *kwnames``: A tuple of the names of the named arguments.
 
 This is implemented by the function pointer type:
-``typedef PyObject *(*vectorcall)(PyObject *callable, Py_ssize_t n, PyObject** args, PyTupleObject *kwnames);``
+``typedef PyObject *(*vectorcallfunc)(PyObject *callable, Py_ssize_t n, PyObject *const *args, PyObject *kwnames);``
 
-Changes to the ``PyTypeObject``
--------------------------------
+Changes to the ``PyTypeObject`` struct
+--------------------------------------
 
-The unused slot ``printfunc tp_print`` is replaced with ``tp_vectorcall_offset``. It has the type ``uintptr_t``.
+The unused slot ``printfunc tp_print`` is replaced with ``tp_vectorcall_offset``. It has the type ``uint32_t``.
```

Comment:
I presume that the change of type from `uintptr_t` to `uint32_t` is a mistake and should be `Py_ssize_t`?

Reply:
Right, thanks for catchnig that.

Reply:
So this discussion is resolved then?